README.md

```diff
@@ -72,6 +73,8 @@ FLAGS
   --only=CONNECTION_ID        Filter only a single connection.
   --skip-migration            Skip running migrations.
   --dry-run                   Dry run migrations and synchronization.
+  --create                    Flag to make create migration. (e.g. `make <FILE_NAME> --create`)
+  --object-name=OBJECT_NAME   Object name to interpolate in template file. (e.g. `make <FILE_NAME> --object=OBJECT_NAME`)
```

Comment:
This is slightly confusing `--object-name=NAME` is a flag only for the `make` command. How do we make that apparent in the docs?

```suggestion
  --object-name=NAME   Object name to interpolate in template file. (e.g. `make <FILENAME> --object=OBJECT_NAME`)
```

Reply:
We need to change the documentation a bit. Let's check how others do it as well. Every command could have their own flags.